docs/api/sql/Raster-operators.md

```diff
@@ -32,6 +32,26 @@ Output:
 IndexOutOfBoundsException: Specified pixel coordinates (6, 2) do not lie in the raster
 ```
 
+### RS_PixelAsPolygon
+
+Introduction: Returns a polygon geometry that bounds the specified pixel.The pixel coordinates specified are 1-indexed.
+
```

Comment:
This documentation needs to be updated to reflect that even if colX and rowY are out of bounds for the raster, they are interpolated assuming the same skew and translate values